src/saga/login.js

```diff
@@ -179,24 +184,23 @@ function* autoConnectCheck (action) {
     } else if (vm && vm.id && vm.status !== 'down') {
       const internalVm = Api.vmToInternal({ vm })
       yield put(updateVms({ vms: [internalVm] }))
-
       yield downloadVmConsole(downloadConsole({ vmId }))
     }
   }
 }
 
-function* initialLoad () {
+function* initialLoad () { // NOTE: This is an in-order fetch, could be made parallel?
```

Comment:
It is: https://redux-saga.js.org/docs/api/#putaction

New action is dispatched.
And since `rootSaga` is created via [all](https://redux-saga.js.org/docs/api/#alleffects---parallel-effects), it should be executed in parallel. Please report/fix if you see otherwise.

Reply:
Ok, that makes sense.  `put` dispatches an async message to redux that will get reduced into the store sometime in future.

It's all good, I'll take the comment out.